.github/workflows/compile-check.yml

```
@@ -0,0 +1,17 @@
+name: compile-check
+
+on: [push, pull_request]
+
+jobs:
+  compile:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          path: "mods"
+      
+      - name: Compile Scripts
+        uses: ASpoonPlaysGames/squirrel-re-compiler@main
```

Probably wanna make a release on action repo and then use that tagged release here so that pushing things to the action repo does not suddenly break CI here ^^

So like release as `v1.0.0` and then
```suggestion
        uses: ASpoonPlaysGames/squirrel-re-compiler@v1
```
here